Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetVerticalUITest.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  PaymentSheetVerticalUITest.swift
```

Comment:
Please ignore the "Fix vertical mode tests subclassing.." commit - that just moves the vertical mode test methods out into this file but it blows up the diff.